modules/core/src/core-layers/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -124,7 +124,8 @@ export default class SolidPolygonLayer extends Layer {
     const {gl} = this.context;
     this.setState({
       numInstances: 0,
-      IndexType: gl.getExtension('OES_element_index_uint') ? Uint32Array : Uint16Array
+      IndexType:
```

Comment:
The luma.gl [feature detection system](http://uber.github.io/luma.gl/#/documentation/api-reference/has-feature) was designed to work across WebGL versions and extensions. Should work both with feature constants and raw extension names:

```
import {hasFeature, FEATURES} from 'luma.gl';
hasFeature(gl, FEATURES.ELEMENT_INDEX_UINT32);
hasFeature(gl, 'OES_element_index_uint');
```